File: code-recipes/php/test/easyship/CalculateOrderDimensionsRecipeTest.php
<?php

namespace Test\easyship;

use Src\easyship\CalculateOrderDimensionsRecipe;
use Test\util\RecipeTestCase;

class CalculateOrderDimensionsRecipeTest extends RecipeTestCase
{
    protected function setUp(): void
    {
        $this->recipe = new CalculateOrderDimensionsRecipe();
        $this->responses = [
            "orders-getOrderItems",
            "listings-getListingsItem"
        ];
    }
}

File: code-recipes/java/src/test/java/easyship/CalculateOrderDimensionsRecipeTest.java
package easyship;

import util.RecipeTest;

import java.util.List;

public class CalculateOrderDimensionsRecipeTest extends RecipeTest {

    protected CalculateOrderDimensionsRecipeTest() {
        super(
                new CalculateOrderDimensionsRecipe(),
                List.of(
                    "orders-getOrderItems",
                    "catalogitems-getCatalogItem",
                    "listings-getListingsItem"
                )
        );
    }
}
